service/orchestrator/catalogs.go

```diff
@@ -250,5 +274,26 @@ func (svc *Service) loadEmbeddedCatalogs() (catalogs []*orchestrator.Catalog, er
 			}
 		}
 	}
+
 	return
 }
+
+// readFolder returns all json files in the given folder
+func readFolder(folder string) ([]string, error) {
```

Comment:
It looks like a function we may use in other places as well in the future. So for now we can leave it here (unexported) in the orchestrator package - but maybe we put it to internal at some point.

Anyway, I would prefer a more descriptive function name, e.g. `getCatalogs` (if it is only for this use) or `getJSONFiles` / `getJSONFileNames`